src/main/java/world/bentobox/border/Settings.java

```diff
@@ -11,146 +11,82 @@
 @StoreAt(filename = "config.yml", path = "addons/Border")
 public class Settings implements ConfigObject {
 
-    @ConfigComment("Border Configuration file by tastybento")
+    @ConfigComment("Border addon configuration file")
+    @ConfigComment("See the documentation at https://docs.bentobox.world/en/latest/addons/Border/")
     @ConfigComment("")
-    @ConfigComment("This list stores GameModes in which Border addon should not work.")
```

Comment:
I don't agree with these changes. The source of truth for Border settings is this file and it should be self documenting and able to be used stand alone. Please put all the comments back.